### DISCUSSION TITLE
module discord.application_commands' missing

### DISCUSSION THREAD

**Discussion by ChaosEmpire** [Rapptz/discord.py#8108] [General]
Hi

i m using a pythin script since a while, and jsut upgraded discord2.0
dont know whcih version i had before
but it was working fine

now i get

from discord.application_commands import ApplicationCommand, option, ApplicationCommandOptionChoice
ModuleNotFoundError: No module named 'discord.application_commands'

did this module got removed?

**Comment by rtk-rnjn** [comment]
It's not `application_commands` but `app_commands`. IIRC it was never `application_commands`, you might be using some other API wrapper other than `discord.py`

**Reply by ChaosEmpire** [reply]
i m using this since several month
it worked very good, until i saw there was an app update and that way i also updated requirements, and i dont know whick  version it has been before...at least now iots no longer working

Discord 2.0 api
pip3 install git+https://github.com/Rapptz/discord.py.git

pip3 install -U git+https://github.com/Rapptz/discord.py

**Reply by XuaTheGrate** [reply]
It never has been ``application_commands``, it was always ``app_commands``. You must have been using a different library.

**Reply by lol219** [reply]
Yes,
It's `app_commands`not `application_commands`
It is located from `discord.app_commands`

**Reply by ChaosEmpire** [reply]
i found a pip3 freeze file i saved a while ago

discord.py==2.0.0a3575+g45d498c1

this was working
i just dont know how to install it again

**Comment by Rapptz** [comment]
`discord.application_commands` never existed in discord.py, it has always been `discord.app_commands`.
